actions/kexec/v1/cmd/kexec.go

```diff
@@ -34,6 +38,30 @@ var kexecCmd = &cobra.Command{
 		initrdPath := os.Getenv("INITRD_PATH")
 		cmdLine := os.Getenv("CMD_LINE")
 
+		//Uefi configuration - more debug that you can imagine
+		uefiBoot := os.Getenv("UEFI_BOOT")
+		uefiboot.UEFIDebug = os.Getenv("UEFI_DEBUG")
+		uefiboot.UEFIBase = os.Getenv("UEFI_BASE")
+		uefiboot.UEFISerialAddr = os.Getenv("UEFI_SERIALADDR")
+		uefiboot.UEFISerialWidth = os.Getenv("UEFI_SERIALWIDTH")
+		uefiboot.UEFISerialHertz = os.Getenv("UEFI_SERIALHERTZ")
+		uefiboot.UEFISerialBaud = os.Getenv("UEFI_SERIALBAUD")
+
+		uefiEnable, _ := strconv.ParseBool(uefiBoot)
+
+		if uefiEnable {
+			// We never come back from here
+			uefiboot.Boot()
+		}
```

Comment:
I think this logic branch could be reorganized a bit to improve readability and avoid the similar variable-package naming. Depending on whether or not you expect `UEFI_BOOT` to be set, or contain some variation of `true` or `false`, you could do:

```go
doUEFIBoot, _ := strconv.ParseBool(os.Getenv("UEFI_BOOT"))
if doUEFIBoot {
    // We never come back from here
    uefiboot.Boot()
}
```

... or if you require the environment variable to be set:

```go
doUEFIBoot, err := strconv.ParseBool(os.Getenv("UEFI_BOOT"))
if err != nil {
    log.Fatalf("please set 'UEFI_BOOT' to 'true' or 'false' - %s", err)
}

if doUEFIBoot {
    // We never come back from here
    uefiboot.Boot()
}
```